ibm_catalog.json

```diff
@@ -5,10 +5,201 @@
             "label": "Power infrastructure for deployable architectures",
             "flavors": [
                 {
-                    "label": "PowerVS workspace",
-                    "name": "standard",
-                    "working_directory": "examples/ibm-catalog/standard-solution",
-                    "usage_template": "module \"power-infrastructure\" {\n  source =                     \"https://cm.globalcatalog.cloud.ibm.com/api/v1-beta/offering/source//${{workingDirectory}}?archive=tgz&flavor=${{flavor}}&kind=${{kind}}&name=${{name}}&version=${{version}}\"\n  prerequisite_workspace_id   = var.prerequisite_workspace_id\n  powervs_zone                = var.powervs_zone\n  powervs_resource_group_name = var.powervs_resource_group_name\n  ssh_private_key             = var.ssh_private_key\n  ibmcloud_api_key            = var.ibmcloud_api_key\n}",
+                    "label": "PowerVS Workspace incl. Secure Landing Zone",
```

Comment:
@kbiegert - name is considered to be temporary. please inform if we get a switch between full-stack and extension variations